yarn-project/circuit-types/src/messaging/inbox_leaf.ts

```diff
@@ -0,0 +1,26 @@
+import { Fr } from '@aztec/circuits.js';
+import { toBigIntBE } from '@aztec/foundation/bigint-buffer';
+import { BufferReader, serializeToBuffer } from '@aztec/foundation/serialize';
+
+export class InboxLeaf {
+  constructor(
+    /** L2 block number in which the message will be included. */
+    public readonly blockNumber: bigint,
+    /** Index of the leaf in L2 block message subtree. */
+    public readonly index: bigint,
+    /** Leaf of the subtree. */
```

Comment:
Leaf in the subtree? When saying leaf of the subtree I for some reason thing the root 😬

Reply:
Addressed